receiver/vcenterreceiver/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/golden v0.137.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest v0.137.0
 	github.com/stretchr/testify v1.11.1
-	github.com/vmware/govmomi v0.50.0
+	github.com/vmware/govmomi v0.52.0
```

Comment:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/43155 skips 0.51.0 , not sure if 0.52.0 is fine?